<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>base_K3bSetup2</class>
  <widget class="QWidget" name="Form1" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>450</height>
      </rect>
    </property>
    <layout class="QHBoxLayout" >
      <property name="margin" >
        <number>0</number>
      </property>
      <item>
        <layout class="QVBoxLayout" >
          <item>
            <widget class="Q3GroupBox" name="groupBox3" >
              <property name="title" >
                <string>Settings</string>
              </property>
              <layout class="QVBoxLayout" >
                <item>
                  <layout class="QHBoxLayout" >
                    <item>
                      <widget class="QCheckBox" name="m_checkUseBurningGroup" >
                        <property name="text" >
                          <string>Use burning group:</string>
                        </property>
                        <property name="whatsThis" stdset="0" >
                          <string>&lt;p>If this option is checked, only the users in the specified group will be able to burn CDs and DVDs, since only they will have access to the devices and the CD recording programs used by K3b.&lt;/p>
&lt;p>Otherwise all users on the system have access to the devices and to all K3b functionality.</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QLineEdit" name="m_editBurningGroup" >
                        <property name="enabled" >
                          <bool>false</bool>
                        </property>
                        <property name="text" >
                          <string>burning</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
                <item>
                  <layout class="QHBoxLayout" >
                    <item>
                      <spacer name="spacer1" >
                        <property name="sizeHint" >
                          <size>
                            <width>20</width>
                            <height>10</height>
                          </size>
                        </property>
                        <property name="sizeType" >
                          <enum>QSizePolicy::Fixed</enum>
                        </property>
                        <property name="orientation" >
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </spacer>
                    </item>
                    <item>
                      <widget class="QLabel" name="textLabel2" >
                        <property name="text" >
                          <string>Users allowed to burn (separated by space):</string>
                        </property>
                        <property name="wordWrap" >
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QLineEdit" name="m_editUsers" >
                        <property name="enabled" >
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <widget class="Q3GroupBox" name="groupBox1" >
              <property name="title" >
                <string>Devices</string>
              </property>
              <layout class="QVBoxLayout" >
                <item>
                  <widget class="QLabel" name="textLabel1_2" >
                    <property name="text" >
                      <string>Check the devices whose permissions you want to be changed</string>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="K3ListView" name="m_viewDevices" >
                    <column>
                      <property name="text" >
                        <string>Device</string>
                      </property>
                      <property name="clickable" >
                        <bool>true</bool>
                      </property>
                      <property name="resizable" >
                        <bool>true</bool>
                      </property>
                    </column>
                    <column>
                      <property name="text" >
                        <string>Devicenode</string>
                      </property>
                      <property name="clickable" >
                        <bool>true</bool>
                      </property>
                      <property name="resizable" >
                        <bool>true</bool>
                      </property>
                    </column>
                    <column>
                      <property name="text" >
                        <string>Permissions</string>
                      </property>
                      <property name="clickable" >
                        <bool>true</bool>
                      </property>
                      <property name="resizable" >
                        <bool>true</bool>
                      </property>
                    </column>
                    <column>
                      <property name="text" >
                        <string>New Permissions</string>
                      </property>
                      <property name="clickable" >
                        <bool>true</bool>
                      </property>
                      <property name="resizable" >
                        <bool>true</bool>
                      </property>
                    </column>
                  </widget>
                </item>
                <item>
                  <layout class="QHBoxLayout" >
                    <item>
                      <spacer name="spacer2" >
                        <property name="sizeHint" >
                          <size>
                            <width>40</width>
                            <height>20</height>
                          </size>
                        </property>
                        <property name="sizeType" >
                          <enum>QSizePolicy::Expanding</enum>
                        </property>
                        <property name="orientation" >
                          <enum>Qt::Horizontal</enum>
                        </property>
                      </spacer>
                    </item>
                    <item>
                      <widget class="QPushButton" name="m_buttonAddDevice" >
                        <property name="text" >
                          <string>Add Device...</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </item>
              </layout>
            </widget>
          </item>
          <item>
            <widget class="Q3GroupBox" name="groupBox2" >
              <property name="title" >
                <string>External Programs</string>
              </property>
              <layout class="QVBoxLayout" >
                <item>
                  <widget class="QTabWidget" name="tabWidget2" >
                    <widget class="QWidget" name="tab" >
                      <attribute name="title" >
                        <string>Found Programs</string>
                      </attribute>
                      <layout class="QVBoxLayout" >
                        <item>
                          <widget class="QLabel" name="textLabel1" >
                            <property name="text" >
                              <string>Check the programs whose permissions you want to be changed</string>
                            </property>
                            <property name="wordWrap" >
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="K3ListView" name="m_viewPrograms" >
                            <column>
                              <property name="text" >
                                <string>Program</string>
                              </property>
                              <property name="clickable" >
                                <bool>true</bool>
                              </property>
                              <property name="resizable" >
                                <bool>true</bool>
                              </property>
                            </column>
                            <column>
                              <property name="text" >
                                <string>Version</string>
                              </property>
                              <property name="clickable" >
                                <bool>true</bool>
                              </property>
                              <property name="resizable" >
                                <bool>true</bool>
                              </property>
                            </column>
                            <column>
                              <property name="text" >
                                <string>Path</string>
                              </property>
                              <property name="clickable" >
                                <bool>true</bool>
                              </property>
                              <property name="resizable" >
                                <bool>true</bool>
                              </property>
                            </column>
                            <column>
                              <property name="text" >
                                <string>Permissions</string>
                              </property>
                              <property name="clickable" >
                                <bool>true</bool>
                              </property>
                              <property name="resizable" >
                                <bool>true</bool>
                              </property>
                            </column>
                            <column>
                              <property name="text" >
                                <string>New Permissions</string>
                              </property>
                              <property name="clickable" >
                                <bool>true</bool>
                              </property>
                              <property name="resizable" >
                                <bool>true</bool>
                              </property>
                            </column>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                    <widget class="QWidget" name="tab" >
                      <attribute name="title" >
                        <string>Search Path</string>
                      </attribute>
                      <layout class="QVBoxLayout" >
                        <item>
                          <widget class="KEditListBox" name="m_editSearchPath" >
                            <property name="title" >
                              <string>Search Path</string>
                            </property>
                            <property name="flat" >
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QLabel" name="textLabel1_3" >
                            <property name="text" >
                              <string>&lt;qt>&lt;b>Hint:&lt;/b> to force K3b to use another than the default name for the executable specify it in the search path.&lt;/qt></string>
                            </property>
                            <property name="wordWrap" >
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                      </layout>
                    </widget>
                  </widget>
                </item>
              </layout>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>K3ListView</class>
      <extends>Q3ListView</extends>
      <header location="global" >k3listview.h</header>
    </customwidget>
    <customwidget>
      <class>KEditListBox</class>
      <extends>Q3ListBox</extends>
      <header location="global" >keditlistbox.h</header>
    </customwidget>
  </customwidgets>
  <includes>
    <include location="local" >k3listview.h</include>
    <include location="local" >keditlistbox.h</include>
    <include location="local" >klineedit.h</include>
  </includes>
  <connections>
    <connection>
      <sender>m_checkUseBurningGroup</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_editBurningGroup</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
    <connection>
      <sender>m_checkUseBurningGroup</sender>
      <signal>toggled(bool)</signal>
      <receiver>m_editUsers</receiver>
      <slot>setEnabled(bool)</slot>
    </connection>
  </connections>
</ui>
